src/schema/nmdc.yaml

```diff
@@ -4,15 +4,12 @@ title: NMDC Schema
 notes:
   - not importing any MIxS terms where the relationship between the name (SCN) and the id isn't 1:1
 description: >-
-  Schema for National Microbiome Data Collaborative (NMDC).
-
-  This schema is organized into multiple modules, such as:
-
-   * a set of core types for representing data values
-   * a subset of the mixs schema
-   * an annotation schema
-   * the NMDC schema itself, into which the other modules are imported
-
+  The NMDC Schema is a foundational framework designed to standardize metadata for the National Microbiome Data 
+  Collaborative (NMDC) and collaborating data providors. By establishing a structured approach to metadata, the NMDC Schema enables researchers to organize, 
+  share, and interpret complex datasets with consistency and clarity. The NMDC Schema is critical substrate used to facilitate 
+  interoperability and collaboration, as it provide a common language for data exchange across systems and disciplines. 
+  In the context of the NMDC, this schema supports the integration of microbiome data from medicine, agriculture, 
+  bioenergy, and environmental science into a cohesive platform.
```

Comment:
This is much better. I agree with all of the points that are made in the new paragraph. "foundational framework" and "critical substrate" sound a little jargony to me, but I don't think that justifies a lot of extra work in rewording.